inc/saitypes.h

```diff
@@ -828,6 +828,9 @@ typedef enum _sai_acl_stage_t
     /** Pre-ingress Stage */
     SAI_ACL_STAGE_PRE_INGRESS,
 
+    /** Post Ingress Stage */
+    SAI_ACL_STAGE_POST_INGRESS,
+
```

Comment:
As discussed in the meeting, we should:

1.   add the switch attribute SAI\_SWITCH\_ATTR\_ACL\_STAGE\_INGRESS so that user can find the capability
2.  Also sai\_acl\_capability\_t should have new fields:
    1.  bool is\_exact\_match\_supported
    2.  bool is\_mix\_exact\_and\_ternary\_supported

Reply:
Thanks for the suggestion. Added SAI_SWITCH_ATTR_ACL_STAGE_INGRESS and also added an enum to query the supported capability as part of PR 1730 which adds sai_acl_supported_match_type_t